examples/allocation-endpoint/terraform/variable.tf

```diff
@@ -20,7 +20,7 @@ variable "project_id" {
 variable "ae_proxy_image" {
   type        = string
   description = "The docker image of the allocation proxy."
-  default     = "us-docker.pkg.dev/agones-images/examples/allocation-endpoint-proxy:0.5"
```

Comment:
The version here should only be bumped until the new image has been pushed to the registry.

Reply:
This image has been successfully built and pushed to my project. Do we need to test it locally?🤔

Reply:
@gongmax, if you approve this PR, I will update the latest tag in production🙂

Reply:
Well, ideally we should only update the version here after the new image has been pushed to agones registry. I.e., the steps should be: 1. bump the version in the makefile. 2. build and push the new image. 3. update the image version where it is used, eg this variable.tf file. 

I can approve this this time, but please push the image to agones registry ASAP.

Reply:
It's done. https://screenshot.googleplex.com/QJjFMME7qVUxMEw

I was under the impression that I had to build and push after receiving approval. Thank you for clarifying.🙂